### ISSUE TITLE
Bumb pypi version?

### ISSUE THREAD

**Issue by someone1** [GoogleCloudPlatform/appengine-pipelines#59]
There were a number of fixes since the last release that should be of use to developers. I know at least our team requires the target propagation fix.

Thank you.

**Comment by someone1** [comment]
It actually looks like that PR was reversed? Was there a reason for this?

**Comment by sit** [comment]
@tkaitchuck @AngryBrock

**Comment by tkaitchuck** [comment]
The three CLs that were reverted were:

https://github.com/GoogleCloudPlatform/appengine-pipelines/commit/633c9fb7f75d506e3ccbd42a9775d3f54195540d

https://github.com/GoogleCloudPlatform/appengine-pipelines/commit/bd9fa1e7000dc6af6fe704137cc53fa6d8cde427

https://github.com/GoogleCloudPlatform/appengine-pipelines/commit/9c6cb797a4e1dff369fc7be220290a49f7b95b8e

I reverted them because they broke the tests. If we can get a version of these in which the tests pass, then there should be no issue, and we can move forward. I'd bet the UI change in isolation could go in without issue. I suspect there is a bug in the other change.
